src/main/java/com/coreos/jetcd/ClientBuilder.java

```diff
@@ -14,13 +14,17 @@
 import io.grpc.NameResolver;
 import io.grpc.netty.GrpcSslContexts;
 import io.netty.handler.ssl.SslContext;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Arrays;
 import java.util.List;
 
 /**
  * ClientBuilder knows how to create an Client instance.
  */
 public class ClientBuilder implements Cloneable {
-
+  private static final List<String> SCHEMES = Arrays.asList("http", "https", "unix", "unixs");
```

Comment:
etcd go client supports unix and unixs mainly for testing purpose i think. Probably no need to support that for java client for now. 

cc/ @xiang90